test-network/scripts/deployCC.sh

```diff
@@ -185,7 +185,7 @@ queryCommitted() {
     peer lifecycle chaincode querycommitted --channelID $CHANNEL_NAME --name fabcar >&log.txt
     res=$?
     set +x
-		test $res -eq 0 && VALUE=$(cat log.txt | grep -o '^Version: [0-9], Sequence: [0-9], Endorsement Plugin: escc, Validation Plugin: vscc')
+		test $res -eq 0 && VALUE=$(cat log.txt | grep -o '^Version: [0-9]\{1,\}, Sequence: [0-9]\{1,\}, Endorsement Plugin: escc, Validation Plugin: vscc')
```

Comment:
checkout how the version is not a string, so this will not be needed, however the sequence is an integer, so maybe we could just use [0-9]+ to catch ints of all sizes ?

Reply:
In resolving a conflict with master I made the change @harrisob suggested and successfully tested it deploying a chaincode all the way to v10.0 s10, and even v11.0 s11 for good measure. :-)